test/cp.js

```diff
@@ -757,7 +757,7 @@ test('should not overwrite recently created files (not give error no-force mode)
   t.is(shell.cat(`${t.context.tmp}/file1`).toString(), 'test1');
 });
 
-test('should copy fifos contents', t => {
+test('should copy fifo contents', t => {
```

Comment:
> There's a conflict between "act like cp" and "pull all the unix out" :-s

I agree these goals are in conflict. I think in this case we should try to stay internally consistent. We should output a warning when copying any fifos and delegate to the plugin to provide actual posix behavior.

If the plugin aims for posix compatibility, what arguments does it need passed? It sounds like it needs `options.recursive`, `srcfile`, & `destFile`, is that correct?